/hik_camera:
  ros__parameters:
    left_camera_name: left_camera

    left_exposure_time: 6000
    left_gain: 16.0

    right_camera_name: right_camera

    right_exposure_time: 6000
    right_gain: 16.0
